org.bndtools.templating.gitrepo/src/org/bndtools/templating/jgit/GitRepoPreferences.java

```diff
@@ -8,10 +8,11 @@
 import org.osgi.framework.FrameworkUtil;
 
 import aQute.bnd.header.Parameters;
+import aQute.bnd.osgi.About;
 
 public class GitRepoPreferences {
 
-    public static final String INITIAL_GITHUB_REPOS = "osgi/workspace, bndtools/workspace;branch=origin/3.3";
+    public static final String INITIAL_GITHUB_REPOS = String.format("osgi/workspace, bndtools/workspace;branch=origin/%d.%d", About.CURRENT.getMajor(), About.CURRENT.getMinor());
```

Comment:
The same could be said for the template repository and bndtools. Maybe we just hard code it and then update it when there is a reason to?